Level one

godofocean7 opened the pull request:
Just need to add navigating system at the end.

godofocean7 committed:
Level one

godofocean7 committed:
player script updated

godofocean7 committed:
.

godofocean7 committed:
Update player.gd

godofocean7 committed:
Update level_1.gd

godofocean7 committed:
.

godofocean7 committed:
Merge pull request #21 from matthewwarn/main

merge main to level_one

SkyePooley committed:
Resolved error from double-jump activation

Yeoungjun had added code to the player script which depended on a node which only existed in one level. This caused the game to throw an error while loading any other level.

Removed this method from player script and moved the functionality to a script in the node which detects the end of the level. Animation now plays with no errors.

Need to work out why double jump is always active.

Co-Authored-By: godofocean7 <171698278+godofocean7@users.noreply.github.com>

SkyePooley committed:
Merge branch 'main' into level_one

SkyePooley commented on Godot Project/Scenes/Ending/Ending_Clear.tscn:
Good to have! Will make this fancier in sprint two if we have time to make some artwork for it.

SkyePooley commented on Godot Project/Scenes/Ending/Ending_Lost.tscn:
I don't think we will be using this one.

SkyePooley commented on Godot Project/Scenes/Ending/Ending_animation.gd:
Can remove the _process method from this. Otherwise it is just wasting clock cycles.

SkyePooley commented on Godot Project/Scenes/Entities/Player/player.gd:
```diff
@@ -46,6 +46,9 @@ var heart3
 var GRAVITY = ProjectSettings.get_setting("physics/2d/default_gravity")
 
+# Show message once
+var level_one_message_shown = false
```
Need to get rid of this, it is no longer used.
```suggestion

```

SkyePooley commented on Godot Project/Scenes/Levels/Level_1_Test/animation_player2.gd:
Not a good name. I think this script is for the moving platform??
This should be named after the moving platform and placed in a folder next to it. Putting this in the level one folder is confusing and could break things.

SkyePooley replied:
Created an issue for this to keep track.

SkyePooley reviewed (approved):
Overall good progress. I will correct the remaining problems while I refactor it to the new level template. 
It can't be added to the scene tree until this refactor is finished.

SkyePooley committed:
Removed redundant variable for unlock message.

This variable was placed in the player script to control a message that pops up when double jump is unlocked. The popup control script has been moved to its own file so this is  now redundant.
